src/algorithms/randomness/rand-choose.md

```diff
@@ -2,29 +2,11 @@
 
 [![rand-badge]][rand] [![cat-os-badge]][cat-os]
 
-Randomly generates a string of given length ASCII characters with custom
-user-defined bytestring, with [`gen_range`].
+Randomly generates a string of given length ASCII characters with custom user-defined bytestring,
+with [`random_range`].
```

Comment:
In general, I've rewrapped markdown at 100 chars to make it easier to read when editing. I'd recommend choosing a value (likely 100) and adding a markdownlint config file for this.

Reply:
I'd accept a lint for it, I know the feeling about wanting to wrap lines.